src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
   margin: 0;
+
+  --text-color-second: #00acdc;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: 100%;
+  text-transform: uppercase;
+  height: 60px;
```

Comment:
The header height is set explicitly, which meets the task requirement to ensure the header height is set in one place.